### DISCUSSION TITLE
How to keep citation key identical (including conflict renamed keys）on multiple devices?

### DISCUSSION THREAD

**Discussion by jasonandjack1997** [retorquere/zotero-better-bibtex#2698] [Q&A] [answered]
I have a home and work pc and need to write one paper on two computers at different times. What is the best practice to keep both computers have identical citation keys.

for example, mannuly sync the corresponding database file (where is it)
or in the plugin prefs how to solve conflicts and make sure the renamed items have identical keys on different computers?

**Comment by jasonandjack1997** [comment]
I am using MS word btw.

**Comment by retorquere** [comment]
> for example, mannuly sync the corresponding database file (where is it)

That will not work, as BBT ties the citation key to the itemID, and itemIDs for the same item differ across synced PCs.

> or in the plugin prefs how to solve conflicts and make sure the renamed items have identical keys on different computers?

The only way to ensure this is to pin the keys.

**Reply by jasonandjack1997** [reply]
Thanks. If I guessed correctly, I should set a citation key pattern on ONE PC and pin the generated citation keys on that computer and sync to the zotero cloud. Then on ANOTHER computer I should not do anything about pinning a citation key, and only set the naming patern the same?

**Reply by retorquere** [reply, marked as answer]
That's one way to do it, but you can auto-pin on both PCs. If a new item is synced to the other PC, it will already have a pinned citekey.
